src/main/java/org/sagebionetworks/web/server/servlet/filter/CORSFilter.java

```diff
@@ -29,93 +29,23 @@ public class CORSFilter extends OncePerRequestFilter {
     "www",
     "staging",
     "tst",
-    "synapse-staging",
-    "status",
-    "staging-signin",
```

Comment:
In general I removed the staging subdomains for all portals and portal-like apps, to keep the list maintainable. We can add back some/all of them if desired.